samcli/local/docker/container.py

```diff
@@ -219,6 +226,26 @@ def start(self, input_data=None):
         # Start the container
         real_container.start()
 
+    def get_logs_stream(self, follow=True):
+        """
+        Returns the container stdout and stderr streams as single iterator.
+        Parameters
+        ----------
+        follow: bool
+            Determines if we need to follow the previous log stream, or get the log stream from the beginning
```

Comment:
Follow here means if we need to follow on the previous container log stream, or we need to get a new log stream that has all the container logs since it got started.

Mainly, after introducing the warm containers, so we need to follow on the previous log stream, and not to return the log stream since the container was created.

Reply:
So we reattach to the container? Should we use the logs api to fetch the invoke logs instead? https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.logs Seems like that is what we are trying to do here.